### DISCUSSION TITLE
Making progress?

### DISCUSSION THREAD

**Discussion by DanHugoDanHugo** [CodeForVegas/cfv-dotorg#4] [General]
I looked through the project as it is today and it looks like maybe Astro is not enabling progress? 

I had mentioned some experimentalism as far as using Astro to generate static *templates* that could be used to generate server-side pages, sent to a client and then perhaps hydrated based on the Astro mechanisms, I might pursue that just to see if it would be interesting.

As for codeforvegas.org site publication, is this a net positive, or negative? Would it be easier to start off implementing the site with all content as-is as a base, and then slicing it up from there?

**Comment by maxwellkahn** [comment]
I believe the template choice to serve up our information is beneficial, icurrently Nick and I are divvying up the load to go off and focus on specific topics instead of both hunting and pecking through our design choices for the mockup.
